build.assets/Dockerfile-centos7

```diff
@@ -246,11 +246,12 @@ COPY --from=libpcsclite \
     /usr/local/lib/libpcsclite.a \
     /usr/local/lib/
 
-# Download pre-built CentOS 7 assets with clang needed to build BPF tools.
```

Comment:
Can you make this same change in `Dockerfile-centos7-fips` as well? So we don't reference it anywhere anymore.